package.json

```
@@ -12,6 +12,8 @@
     "clean": "rimraf coverage dist tmp",
     "lint": "eslint . --ext .ts",
     "prebuild": "npm run lint",
+    "prettier": "prettier -w --config .prettierrc lib",
+    "prettier:check": "prettier -c --config .prettierrc lib",
```

What about also checking the `docs`, `scripts` and `tests` directories?

---

I have no problem with that. Do you suggest to simply update the command or update `.prettierrc`?

---

Just adding that is enough, and your PR will be ready 👍 

```suggestion
    "prettier": "prettier -w --config .prettierrc {lib,docs,tests}",
    "prettier:check": "prettier -c --config .prettierrc {lib,docs,tests}",
```

---

Final question: should plain `prettier` script auto-fix issues or only check for issues? Accordingly, we should use `prettier:fix` or `prettier:check` for the other.

I have nptoced that `lint` os just a check. We might wamt to be consistant in here.

---

+1 for consistency with `lint`.

Let's go for `prettier` + `prettier:fix`. 
Thanks 🙏

---

```suggestion
    "prettier": "prettier -c --config .prettierrc {lib,docs,tests}",
    "prettier:fix": "prettier -w --config .prettierrc {lib,docs,tests}",
```